### PR TITLE
Link to the resources from the events page

### PR DESCRIPTION
**Pull request by jwforres**
Fixes https://github.com/openshift/origin/issues/7523

### CODE REVIEW

**Comment by jwforres** [comment]
@spadgett PTAL

Should we do the same from the events sidebar? If so should that link go to the events tab for the resource? I would expect clicking on a link in the sidebar to take me to a more detailed view of that event since we clipping them.

File: app/scripts/services/api.js [hunk 1]
```diff
@@ -101,7 +101,10 @@ angular.module('openshiftConsole')
     }
     var parts = apiVersion.split("/");
     if (parts.length === 1) {
-      return {group:'', version: parts[0]};
+      if (parts[0] === "v1") {
+        return {group: '', version: parts[0]};
+      }
+      return {group: parts[0], version: ''};
```
Comment (jwforres) [inline app/scripts/services/api.js]: fyi already discussed this change with @liggitt

File: app/views/directives/events.html [hunk 2]
```diff
@@ -57,7 +57,10 @@
         <td data-title="Time" class="nowrap">{{event.lastTimestamp | date:'mediumTime'}}</td>
         <td ng-if="!resourceKind || !resourceName" data-title="Name">
           <div class="hidden-xs-block visible-sm-block visible-md-block hidden-lg-block">{{event.involvedObject.kind | humanizeKind : true}}</div>
-          {{event.involvedObject.name}}
+          <a
+            ng-href="{{event.involvedObject.name | navigateResourceURL : event.involvedObject.kind : event.metadata.namespace : event.involvedObject.apiVersion}}"
+            ng-if="event.involvedObject.name | navigateResourceURL : event.involvedObject.kind : event.metadata.namespace : event.involvedObject.apiVersion">{{event.involvedObject.name}}</a>
+          <span ng-if="!(event.involvedObject.name | navigateResourceURL : event.involvedObject.kind : event.metadata.namespace : event.involvedObject.apiVersion)">{{event.involvedObject.name}}</span>
```
Comment (spadgett) [inline app/views/directives/events.html]: Worth adding an `ng-init` to the repeat to avoid having this in 3 places?

**Comment by jwforres** [comment]
had to update api service spec tests to match the change in behavior of parseGroupVersion

**Comment by jwforres** [comment]
accidently did this against my master branch, closing and re-opening against a different branch

**Comment by jwforres** [comment]
https://github.com/openshift/origin-web-console/pull/883

**Pull request closed without merging**